src/main/java/org/fundacionjala/coding/reinaldoA/spinWords/movies/Customer.java

```diff
@@ -0,0 +1,56 @@
+package org.fundacionjala.coding.reinaldoA.spinWords.movies;
+
+import java.util.Enumeration;
+import java.util.Vector;
+
+/**
+ * This is the class Customer of rental movie.
+ */
+public class Customer {
+    private String nameCustomer;
+    private Vector rentalsCustomer = new Vector();
+
+    /**
+     * @param nameCustomer String with words.
+     *                     Constructor.
+     */
+    Customer(final String nameCustomer) {
+        this.nameCustomer = nameCustomer;
+    }
+
+    /**
+     * @param arg String with words.
+     */
+    public void addRental(final Rental arg) {
+        this.rentalsCustomer.addElement(arg);
+    }
+
+    /**
+     * @return the nameCustomer string.
+     */
+    public String getName() {
+        return this.nameCustomer;
+    }
+
+    /**
+     * @return the string.
+     */
+    public String statement() {
```

Comment:
you can split this method
for instance calculateTotalAmount, calculateTotalFrequentRenterPoints and statement